### ISSUE TITLE
Issue deploying from BLT into ACSF

### ISSUE THREAD

**Issue by mpifalo** [acquia/blt#2724]
Was referred here by Acquia support. We use BLT, Travis CI to deploy up to ACSF. We see the error noted below.

My system information:
* Operating system type: MacOS
* Operating system version: 10.13.3
* BLT version: 8.9.14

No errors reported in `blt doctor`

The error we are seeing in the failure is here:
```
cURL error 3: <url> malformed (see [error] 
http://curl.haxx.se/libcurl/c/libcurl-errors.html) 
Acquia Connect subscription updated. 
Error occurred while retrieving Acquia subscription information. See [error] 
logs for details. 
[Acquia\Blt\Robo\Tasks\DrushTask] Exit code 1 Time 8.619s 
[Acquia\Blt\Robo\Tasks\DrushTask] Running /var/www/html/ebsco.01dev/vendor/bin/drush cc drush --uri=ssotest.dev-ebsco.acsitefactory.com --yes && 
/var/www/html/ebsco.01dev/vendor/bin/drush ebsco:mum --uri=ssotest.dev-ebsco.acsitefactory.com --yes && 
/var/www/html/ebsco.01dev/vendor/bin/drush cr --uri=ssotest.dev-ebsco.acsitefactory.com --yes in /var/www/html/ebsco.01dev/docroot
```
Acquia support is thinking that `docroot/modules/custom/ebsco_core/ebsco_core.drush.inc` is utilizing drush9 syntax, but is invoking drush 8.

They pointed out this snippet from the file:
```
function ebsco_core_drush_command() {
  $items['ebsco-aquia-connect-update-subscription'] = [
    'description' => 'Update Acquia Connector Subscription and delete search cache.',
    'drupal dependencies' => ['acquia_connector'],
    'aliases' => ['ebsco:acus'],
  ];
```
It's worth noting this is the same drush code we've been using since Oct 31, but only in the past week have we run into issues with BLT and ACSF.

Appreciate any and all thoughts; please let me know if additional information is needed.

**Comment by arknoll** [comment]
@mpifalo To confirm, did you update BLT and then you started seeing this error?

**Comment by mpifalo** [comment]
@arknoll We had already been updated to 8.9.14 for a while (updated Oct 23). We had successfully deployed a few tags since that update.

**Comment by arknoll** [comment]
@mpifalo when manually running the drush commands that are listed in your issue, are you seeing the same curl error occuring?

**Comment by mpifalo** [comment]
@arknoll When I updated to ACSF 1.45 (which was at the same time as updating BLT), and running the manual drush command I did not see that error. Instead, we ran into file permission errors on acsf.settings.php - I needed to chmod that file to have me as the own on my local and then I was able to not have errors.

Unfortunately the timeline is a bit muddy now as twice we thought we had resolved the ACSF issues only to find out we were unable to deploy fully into 01live (but 01test worked). Partway through working with support they had me use the blt command line; we had also installed the BLT patch 2685, and I believe it's at that point that cUrl error was spotted.

In continuing to work with support we are not seeing that error anymore, but instead a different one with db-update.sh.

In case it matters, we are on Drush 8, not 9. Sorry if this doesn't fully answer your question.

**Comment by arknoll** [comment]
@mpifalo I believe you have a fix for the problem you were seeing. Is it ok to close this issue?

**Comment by mpifalo** [comment]
@arknoll Yes, please close. The issue we were having was resolved by [https://github.com/acquia/blt/pull/2730](https://github.com/acquia/blt/pull/2730).
